mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -113,7 +115,7 @@ pub const GETH_DOCKER_IMAGE_WITH_TAG: &str = "docker.io/ethereum/client-go:stabl
 pub const NUCLEUS_IMAGE: &str = "docker.io/komodoofficial/nucleusd";
 pub const ATOM_IMAGE: &str = "docker.io/komodoofficial/gaiad";
 pub const IBC_RELAYER_IMAGE: &str = "docker.io/komodoofficial/ibc-relayer";
-
+pub const SOLANA_CLUSTER_DOCKER_IMAGE: &str = "docker.io/0xbdj/solana-node-test";
```

Comment:
We probably need to push this image under https://hub.docker.com/u/komodoofficial org.